docs/http-clients.md

```diff
@@ -0,0 +1,17 @@
+# HTTP Clients
```

Comment:
This title and file filename shoul have RPC in it.

"HTTP" is too generic as want to avoid confusion with "HTTP Gateway"